tests/Functional/DomainCommandsTest.php

```diff
@@ -29,35 +29,43 @@ class DomainCommandsTest extends TestCase
      */
     public function testDomainAddListLookupRemove()
     {
-        $domainList = $this->terminusJsonResponse(sprintf('domain:list %s', $this->getSiteEnv()));
+        // Running tests against the "test" environment instead of a multidev one since "domain:lookup" command does not
+        // search domains across multidev environments.
+        $siteEnv = sprintf('%s.test', $this->getSiteName());
+        $domainList = $this->terminusJsonResponse(sprintf('domain:list %s', $siteEnv));
         $this->assertIsArray($domainList);
         $this->assertNotEmpty($domainList);
 
         $testDomain = uniqid('test-') . '.test';
-        $this->terminus(sprintf('domain:add %s %s', $this->getSiteEnv(), $testDomain));
+        $this->terminus(sprintf('domain:add %s %s', $siteEnv, $testDomain));
         sleep(10);
-        $domainList = $this->terminusJsonResponse(sprintf('domain:list %s', $this->getSiteEnv()));
+        $domainList = $this->terminusJsonResponse(sprintf('domain:list %s', $siteEnv));
         $domains = array_column($domainList, 'id');
         $this->assertContains($testDomain, $domains, 'Domain list should contain added domain');
 
-        // @fixme CMS-238
-//        $lookUpResult = $this->terminusJsonResponse(sprintf('domain:lookup %s', $newDomain));
-//        $this->assertEquals([], $lookUpResult);
+        $lookUpResult = $this->terminusJsonResponse(sprintf('domain:lookup %s', $testDomain));
+        $this->assertIsArray($lookUpResult);
+        $this->assertNotEmpty($lookUpResult);
+        $this->assertArrayHasKey('site_id', $lookUpResult);
```

Comment:
Kevin found a bug in the `org:list` command, the org name and label were empty.  The test had been checking that the returned array contained the keys, but not that they had an actual value.  Adding `assertNotEmpty` caught the bug.  Would we want to do similar here?

Reply:
Good point, updated